File: Programming-languages-and-laboratory/Python/Lessons/34_20221221/Nutrizionista.py
from Dipendente import Dipendente


class Nutrizionista(Dipendente):
    def __init__(self, codice, tipo, nome, telefono, medico, appuntamenti_settimanali, costo_orario):
        super().__init__(codice, tipo, nome, costo_orario)
        self._telefono = telefono
        self._medico = medico
        self._appuntamenti_settimanali = appuntamenti_settimanali

    def __str__(self):
        return self._nome + "\t" + str(self._codice) + "\t" + self._tipo + "\t-\t-\t" + str(self._telefono) + "\t" + str(self._medico) + "\t" + str(self._appuntamenti_settimanali) + "\t" + str(self._costo_orario)

File: II - Linguaggi di programmazione e laboratorio/Java/Exercises/41_compito_java_2022_11_10/Nutrizionista.java
public class Nutrizionista extends Dipendente {
	private int telefono;
	private boolean medico;
	private int appuntamentiSettimanali;

	public Nutrizionista(int codice, String tipo, String nomeDipendente, float costoOrario, int telefono,
			boolean medico, int appuntamentiSettimanali) {
		super(codice, tipo, nomeDipendente, costoOrario);
		this.telefono = telefono;
		this.medico = medico;
		this.appuntamentiSettimanali = appuntamentiSettimanali;
	}

	public String toString() {
		return "nutri" + "\t" + super.toString() + "\t-\t-\t-\t" + telefono + "\t" + medico + "\t" + appuntamentiSettimanali
				+ "\t" + costoOrario;
	}
}